smart_proxy_dynflow.gemspec

```diff
@@ -21,7 +21,9 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '~> 2.5'
 
-  gem.add_runtime_dependency "logging"
+  gem.add_runtime_dependency('dynflow', "~> 1.1")
+  gem.add_runtime_dependency('rest-client')
```

Comment:
Just a random note: in smart-proxy itself there's also multiple similar implementations for HTTP abstraction. I think it makes sense to consider a common solution. Food for thought.